Sources/TextView/TextView.swift

```diff
@@ -95,9 +95,13 @@ public struct TextView: View {
 		
 		public func updateUIView(_ textView: UITextView, context _: Context) {
 			if !shouldWaitUntilCommit || textView.markedTextRange == nil {
-				let selectedRange = textView.selectedRange
+				let textViewWasEmpty = textView.text.isEmpty
+				var selectedRange = textView.selectedTextRange
 				textView.text = text
-				textView.selectedRange = selectedRange
+				if textViewWasEmpty {
```

Comment:
https://github.com/kenmueller/TextView/pull/21/files#r429013942 makes me think that maybe it's not safe to try to preserve the text range like this in other cases, but at least this fixes the case when the text view is initialized with text.